src/ResourceManagement/Compute/Domain/IVirtualMachineScaleSetVMsBeta.cs

```diff
@@ -36,6 +36,20 @@ public interface IVirtualMachineScaleSetVMsBeta :
         /// <return>A representation of the deferred computation of this call.</return>
         Task DeleteInstancesAsync(string[] instanceIds, CancellationToken cancellationToken = default(CancellationToken));
 
+        /// <summary>
+        /// Get the specified virtual machine instance from the scale set.
+        /// </summary>
+        /// <param name="instanceId">Instance ID of the virtual machine scale set instance to be fetched.</param>
+        /// <returns>The virtual machine scale set instance.</returns>
+        IVirtualMachineScaleSetVM GetInstance(string instanceId);
```

Comment:
@IanCMarshall Thanks for contributing to the repo :)

Would you kindly add tests for the newly added functionality as well?
 Do let me know if you need help with it.